serverless.yml

```diff
@@ -65,6 +65,11 @@ functions:
     timeout: 30
     events:
       - sns: ${env:SNS_TOPIC_ARN}
+  cru-org-feed:
+    handler: scheduled/cru-org-feed.handler
```

Comment:
If you want, you can put `redshift.js` into scheduled as well. I can rebase my testing branch.